src/internal/metrics.ts

```diff
@@ -2,6 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 import { Metrics } from '@cloudscape-design/component-toolkit/internal';
 
-import { PACKAGE_SOURCE, PACKAGE_VERSION } from './environment';
+import { PACKAGE_SOURCE, PACKAGE_VERSION, THEME } from './environment';
 
-export const metrics = new Metrics(PACKAGE_SOURCE, PACKAGE_VERSION);
+export const metrics = new Metrics({ packageSource: PACKAGE_SOURCE, packageVersion: PACKAGE_VERSION, theme: THEME });
```

Comment:
Addressing this follow-up https://github.com/cloudscape-design/component-toolkit/pull/136#discussion_r2153239411

Yes I know this `THEME` value does not support `vr`, but it is fine for this use-case

Reply:
can you explain a bit more?